src/main/content/_assets/js/builds.js

```diff
@@ -1006,7 +1006,7 @@ $(document).ready(function () {
                     setFocusOnModalPopupElement();
                     var build_tool = $('input[name=\'build_system\']:checked').val();
                     if(build_tool == "maven") {
-                        $('#cmd_to_run').text("mvnw liberty:run");
+                        $('#cmd_to_run').text("mvnw liberty:dev");
                     }
                     else {
                         $('#cmd_to_run').text("gradlew libertyStart");
```

Comment:
the Gradle command also needs to be updated to `gradlew libertyDev` to match `mvnw liberty:dev`

Reply:
Additionally, we need to update the README to also use `gradlew libertyDev` and `mvnw liberty:dev`
See https://github.com/OpenLiberty/start.openliberty.io/blob/main/src/main/resources/templates/readme/README.txt

Reply:
ok